From: Richard B Winters <rik@mmogp.com>
Date: Fri, 29 Jan 2016 10:38:09 -0500
Subject: Remove AM_CONDITIONAL for UNAME_IS_DEBIAN

 - The conditional was implemented as part of a 
   resolution for a packaging issue on Debian systems, 
   but is not necessary
 - This patch was forwarded and merged upstream

Change-Id: I3f41fb84c5fabe446154bcc6236703b8294dd832
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index f378f26..1a8de80 100644
--- a/configure.ac
+++ b/configure.ac
@@ -41,8 +41,6 @@ dnl command line
 dnl Wrap in m4_ifdef to avoid breaking on older platforms
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES])
 
-AM_CONDITIONAL([UNAME_IS_DEBIAN], [test x`uname -v | grep -oh Debian` = x"Debian"])
-
 AC_SUBST(OCF_ROOT_DIR)
 AC_SUBST(CRM_DAEMON_USER)
 
